locations/spiders/mydentist_gb.py

```diff
@@ -34,4 +34,12 @@ def inspect_item(self, item, response):
         item["lat"] = re.search(r"\"_readModeLat\":(-?[\d.]+),", response.text).group(1)
         item["lon"] = re.search(r"\"_readModeLon\":(-?[\d.]+),", response.text).group(1)
 
+        # City can come back as eg ["Penistone", "Sheffield"] - put the locality on the end of street address
+        if "city" in item and isinstance(item["city"], list):
+            item["street_address"] = ", ".join([item["street_address"]] + item["city"][:-1])
+            item["city"] = item["city"][-1]
+
+        if item["name"] is not None and (item["name"].startswith("mydentist") or item["name"].startswith("{my}dentist")):
```

Comment:
I'm not attached to this - some seem to be independent like `Ayr Dental Centre`, others are inconsistently `mydentist` or `{my}dentist`. Thoughts?

Reply:
If it really is independent, we'd want to do the opposite. Only set the brand on the others, and leave this one without a brand.

However (2 years ago) it was a [mydentist](https://www.google.com/maps/@55.4621818,-4.6317425,3a,75y,128.11h,89.83t/data=!3m7!1e1!3m5!1sehHcg8hFnwjig3RClkSR_A!2e0!6shttps:%2F%2Fstreetviewpixels-pa.googleapis.com%2Fv1%2Fthumbnail%3Fpanoid%3DehHcg8hFnwjig3RClkSR_A%26cb_client%3Dmaps_sv.tactile.gps%26w%3D203%26h%3D100%26yaw%3D107.218544%26pitch%3D0%26thumbfov%3D100!7i16384!8i8192?entry=ttu), it may just be something wired with there data.

Reply:
![image](https://github.com/alltheplaces/alltheplaces/assets/7499526/f12d2564-4984-4acf-8bd6-b1a194ff70d4)
They seem to prefer `{my}dentist` while acknowledging `mydentist` in more legalese places

Reply:
You inspired me to check - `Ayr Dental Centre` is on their list of practices (which we scrape) but does _not_ show up if you put its postcode into their store finder. 

Do we keep scraping it but not brand it, or discard it?

Reply:
And to confuse it even more: [Stafford Dental Centre](https://www.mydentist.co.uk/dentists/practices/england/west-midlands/stafford/24-eastgate-street) which does show in search results.

It looks like there is 27 without `{my}dentist` or `mydentist`.

<details>
<summary>Links</summary>

```
website=https://www.mydentist.co.uk/dentists/practices/england/east-of-england/chelmsford/victoria-road
website=https://www.mydentist.co.uk/dentists/practices/england/east-of-england/ipswich/199-nacton-road
website=https://www.mydentist.co.uk/dentists/practices/england/east-of-england/norwich/40-48-george-lane
website=https://www.mydentist.co.uk/dentists/practices/england/north-east-england/south-shields/flagg-court-primary-care-centre
website=https://www.mydentist.co.uk/dentists/practices/england/north-east-england/stockton-on-tees/lysander-house
website=https://www.mydentist.co.uk/dentists/practices/england/north-west-england/carnforth/6---8-chapel-lane
website=https://www.mydentist.co.uk/dentists/practices/england/north-west-england/chorley/buckshaw-village-health-centre
website=https://www.mydentist.co.uk/dentists/practices/england/north-west-england/liverpool/abercromby-health-centre
website=https://www.mydentist.co.uk/dentists/practices/england/north-west-england/liverpool/litherland-town-hall
website=https://www.mydentist.co.uk/dentists/practices/england/south-east-england/basingstoke/shakespeare-house-health-centre
website=https://www.mydentist.co.uk/dentists/practices/england/south-east-england/basingstoke/sullivan-road
website=https://www.mydentist.co.uk/dentists/practices/england/south-west-england/bristol/68-70-east-street
website=https://www.mydentist.co.uk/dentists/practices/england/south-west-england/cornwall/66-hayle-terrace
website=https://www.mydentist.co.uk/dentists/practices/england/south-west-england/redruth/28-green-lane
website=https://www.mydentist.co.uk/dentists/practices/england/south-west-england/taunton/kilkenny-house
website=https://www.mydentist.co.uk/dentists/practices/england/south-west-england/tavistock/kilworthy-park
website=https://www.mydentist.co.uk/dentists/practices/england/south-west-england/truro/41-lemon-street-
website=https://www.mydentist.co.uk/dentists/practices/england/south-west-england/truro/forth-coth
website=https://www.mydentist.co.uk/dentists/practices/england/south-west-england/weymouth/bridges-dental-centre
website=https://www.mydentist.co.uk/dentists/practices/england/west-midlands/birmingham/183-redditch-road
website=https://www.mydentist.co.uk/dentists/practices/england/west-midlands/newcastle-under-lyme/1-cherry-orchard
website=https://www.mydentist.co.uk/dentists/practices/england/west-midlands/stafford/24-eastgate-street
website=https://www.mydentist.co.uk/dentists/practices/england/west-midlands/stoke-on-trent/1-penfleet-avenue
website=https://www.mydentist.co.uk/dentists/practices/northernireland/armagh-/armagh/56-kinelowen-street
website=https://www.mydentist.co.uk/dentists/practices/northernireland/belfast-/belfast/152-anderstontown-road
website=https://www.mydentist.co.uk/dentists/practices/scotland/ayrshire/south-ayrshire/10-arthur-street-
website=https://www.mydentist.co.uk/dentists/practices/wales/west-wales/aberystwyth/park-avenue
```
</details>

I checked [one more](https://www.google.com/maps/@50.1885867,-5.418614,3a,48.9y,141.06h,88.01t/data=!3m7!1e1!3m5!1sN1OApM2v2Wmib22ZE1zSQw!2e0!6shttps:%2F%2Fstreetviewpixels-pa.googleapis.com%2Fv1%2Fthumbnail%3Fpanoid%3DN1OApM2v2Wmib22ZE1zSQw%26cb_client%3Dmaps_sv.tactile.gps%26w%3D203%26h%3D100%26yaw%3D241.08803%26pitch%3D0%26thumbfov%3D100!7i16384!8i8192?entry=ttu), and it's from May, not branded.

> Do we keep scraping it but not brand it, or discard it?

My vote is to collect it, but don't brand it. So to do this, we need to remove `item_attributes` (they get applied to every POI) and manually apply the `brand` and `brand_wikidata` for most the POIs.

I typically do this with:
```
    MYDENTIST = {"brand": "My Dentist", "brand_wikidata": "Q65118035"}
```
and 
```
        item.update(self.MYDENTIST)
```

Reply:
Thank you